addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyScanRule.java

```diff
@@ -557,6 +571,18 @@ private AlertBuilder buildStyleUnsafeHashAlert(String param, String evidence) {
                                 MESSAGE_PREFIX + "stylesrc.unsafe.hashes.refs"));
     }
 
+    private AlertBuilder buildScriptUnsafeEvalAlert(String param, String evidence) {
+        return getBuilder(
+                        Constant.messages.getString(MESSAGE_PREFIX + "scriptsrc.unsafe.eval.name"),
+                        "9")
```

Comment:
This should be a separate alert ref since it has its own name etc.

Reply:
My mistake. I didn't notice that there was already alert ref '9'.
Will be changed to '10'.